src/main/java/StringManipulation.java

```diff
@@ -1,33 +1,134 @@
 public class StringManipulation implements StringManipulationInterface {
 
+    private String string;
+
     @Override
     public String getString() {
-        return null;
+        return string;
     }
 
     @Override
     public void setString(String string) {
+        this.string = string;
     }
 
     @Override
     public int count() {
-        return 0;
+        if (string == null)
+            return 0;
+        if (string.length() == 0)
+            return 0;
+
+        int count = 0;
+        for (int i = 0; i < string.length() - 2; i++) {
+            if (i == 0 && !isSpaceChar(string.charAt(i)))
+                count++;
+            else if (isSpaceChar(string.charAt(i)) && !isSpaceChar(string.charAt(i + 1)))
+                count++;
+        }
+
+        return count;
```

Comment:
Pretty laborious solution, I like the meticulousness and I see the hussle, but like I say in a later comment you can use the split method of the string class with the regex "\\s+", giving you a string array of everything split by whitespaces, and with that you could just return the length of that generated array, easy peasy.